Source/Shared/BPConfiguration.m

```diff
@@ -70,6 +70,10 @@
         "Exclude a testcase in the set of tests to run (takes priority over `include`)."},
     {'X', "xcode-path", required_argument, NULL, BP_VALUE | BP_PATH, "xcodePath",
         "Path to xcode."},
+    {'u', "use-simulator", required_argument, NULL, BP_VALUE, "useDeviceID",
+        "The device UUID of simulator to reuse. (BP INTERNAL USE ONLY). "},
+    {'D', "delete-simulator", required_argument, NULL, BP_VALUE, "delDeviceID",
```

Comment:
We should do UDID since it's what apple uses. ID is kind of ambiguous because it could be taken to mean the _name_ of the simulator (e.g. BPXXX in our case).